src/commands/build.cr

```diff
@@ -34,7 +34,7 @@ module Shards
         Shards.logger.debug "crystal #{args.join(' ')}"
 
         error = IO::Memory.new
-        status = Process.run("crystal", args: args, output: error, error: error)
+        status = Process.run("crystal", args: args, output: STDOUT, error: error)
```

Comment:
Can you change to `output: Redirect::Inherit`? Then I'll merge, thanks!

Reply:
Done! :tada: